### PR TITLE
Add timeout, max_retry while checking redfish availability

### PR DESCRIPTION
**Pull request by dashmage**

### CODE REVIEW

File: src/hw_tools.py [hunk 1]
```diff
@@ -337,7 +337,9 @@ def redfish_available() -> bool:
     host = f"https://{bmc_address}"
     try:
         # credentials can be empty because we're only checking if redfish service is accessible
-        redfish_obj = redfish_client(base_url=host, username="", password="")
+        redfish_obj = redfish_client(
+            base_url=host, username="", password="", timeout=3, max_retry=2
```
Comment (rgildein) [inline src/hw_tools.py:341]: I think that it would be better if timeout and max_retry are define as constants instead of using values like this.

Comment (dashmage) [reply src/hw_tools.py]: Fixed this in latest commit :)

**Review by rgildein — APPROVED** [review]
LGTM

**Review by jneo8 — APPROVED** [review]
LGTM

**Pull request merged**